changelog/@unreleased/pr-3068.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: improvement
+improvement:
+  description: Baseline no longer explicitly sets the archive task duplicates strategy
+    to `WARN`. Archive tasks will now use the Gradle default behavior which will fail
+    if there are duplicates files and a duplicates strategy was not explicitly set.
```

Comment:
Isn't this just going to prevent baseline upgrading on a whole load of repos?

Reply:
I would hope not. If it does, we almost certainly want to know about it because it means those builds aren't reproducible.

Reply:
I think this is a pretty common issue - I've definitely seen the warning a lot. Even if it's bad, we can't block baseline upgrades for this kind of fix if the issue is widespread. It's a shame we can't easily find which repos this is happening in, then we could decided whether to do feature flag or just go for it.

Fwiw, sls-packging sets up it's own behaviour for `distTar`s:
* https://github.com/palantir/sls-packaging/blob/548f03e7cb2918f21af6e0d3a6c16402d553c1ac/gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/DistTarTask.java#L109
* https://github.com/palantir/sls-packaging/blob/0345a84afa5bad80c01bb24555a8925d3609a61b/gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/asset/AssetDistributionPlugin.java#L95

Reply:
I guess we could do an RC then a "background excavator" to find out how many failures there are before merging it in

Reply:
Ack, I assumed this was not widespread.